lib/datadog/core/environment/variable_helpers.rb

```diff
@@ -0,0 +1,43 @@
+require 'ddtrace/ext/environment'
+
+module Datadog
+  module Core
+    # Namespace for handling application environment
+    module Environment
+      # Defines helper methods for environment
+      module VariableHelpers
```

Comment:
This module was previously the only resident of `datadog/environment.rb`. I decided to move it here as well, as not not have future confusion around what "environment" means in `ddtrace`.

Reply:
Minor: I think you meant `lib/ddtrace/environment.rb` (took me a bit to find it)

Reply:
Yes, indeed, updated ty!